style.scss

```
@@ -0,0 +1,346 @@
+                color: $cBrown;
+                font-size: 25px;
+            }
+
+            & h4 {
+                margin-top: 15px;
+                color: $cBrown;
+                font-size: 25px;
+                text-align: right;
+            }
+
+            & h5 {
+                margin-top: 15px;
+                font-size: 10px;
+                text-align: center;
+            }
+
+            & h6 {
+                display: none;
+            }
+        }
+    }
+
+    &-collections {
+
+        display: flex;
+        flex-wrap: wrap;
+        gap: 5.1%;
+        margin-left: 38px;
+        margin-top: 30px;
+
+        & img {
+            border: 1px solid $cBrown;
+            width: 100px;
+            height: 100px;
+            margin-top: 15px;
+        }
+
+        & h6 {
+            width: 100px;
+            height: 100px;
+            background-color: $cBrown;
+            margin-top: 15px;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            color: white;
+            font-size: 10px;
+        }
+    }
+}
+
+}
+
+
+
+
+
+
+
+// =========================== desk top=================================================
+@media screen and (min-width:1030px) {
+// ========================= top page ====================
+
+body {
+    position: relative;
+}
+
+.page1 {
+    height: 100vh;
+
+    .oneArea {
+        position: absolute;
+        top: 0;
+        right: 0;
+        height: 100vh;
+        width: 35vw;
+        background-color: $cBrown;
+    }
+
+    .top {
+
+        & img {
+            position: absolute;
+            top: 10vh;
+            left: 45%;
+            width: 550px;
+        }
+
+        &-text {
+            position: absolute;
+            top: 30vh;
+            left: 100px;
+
+            & h1 {
+                font-size: 34px;
+            }
+
+            & p {
+                font-size: 18px;
+                top: 0;
+                left: 0;
+            }
+        }
+    }
+
+   
+}
+
+
+// ============================= New collections =====================
+.new {
+
+    margin-top: 110px;
+
+    & h2 {
+        width: 328px;
+        font-size: 34px;
+        text-align: right;
+        padding-right: 20px;
+        border-bottom: 1px solid $cBrown;
+        }
+
+    & img {
+    border: 1px solid $cBrown;
+    width: 350px;
+    }
+
+    
+    &-collections {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 5.1%;
+        margin-left: 8.9%;
+        margin-top: 90px;
+        
+        & h6 {
+            font-size: 34px;
+            text-align: right;
+            margin-right: 130px;
+        }
+    }
+}
+
+// ======================== Best collections =====================
+.best {
+    position: relative;
+
+    & h2 {
+        width: 328px;
+        font-size: 34px;
+        text-align: right;
+        padding-right: 20px;
+        border-bottom: 1px solid $cBrown;
+    }
+
+    &-top {
```

This is suggestion. In your ways, `display: flex;` is not working because of `position: absolute;`.

```suggestion
    &-top {
       margin-top: 100px;
       display: flex;
       justify-content: space-evenly;
         
         


         & img {
             width: 500px;
             height: 500px;
             border: 1px solid $cBrown;
         }

         &-text {
            width: 40%;
            padding: 50px 20px;

             & .best-text-left {
                 font-size: 50px;
                 color: $cBrown;
                 
             }
             & .best-text-right {
                 font-size: 50px;
                 color: $cBrown;
                 text-align: right;
                 margin: 20px 0px;
             }
             & p {
                 font-size: 18px;
                 text-align: center;
                 margin-top: 30px;
             }
             & h6 {
                 font-size: 34px;
                 margin-top: 3px;
                 text-align: right;
             }
         }
     }
```

---

Fixed! It got much better, thank you for correcting a lot..